top/templates/strings/pt_strings.yaml

```diff
@@ -0,0 +1,114 @@
+#    https://github.com/hatnote/top/issues
+#------------------------------------------------------------------------------
+
+#-----------
+# Page title
+#------------------------------------------------------------------------------
+
+title: Top 100
+full_title: >-
+ Top 100 da Wikipédia em Português em {formatted_date}
+
+index_title: >-
+  Top 100 da Wikipédia em Português
+
+rss_title: >-
+  Os 100 artigos mais visitados na Wikipédia em Português
+
+wikipedia: Wikipedia
+language_project: Wikipédia em Português
+tagline: >-
+    Os artigos mais visitados da Wikipédia em português, atualizados diariamente.<a
+    href="#about" class="learn">Ler mais</a>...
+
+alt_tagline: >-
+     Os artigos mais visitados da Wikipédia em português, atualizados diariamente.
+
+#------
+# Links
+#------------------------------------------------------------------------------
+
+about_link: Sobre
+more_langs_link: Outros idiomas
+feed_link: Feed
+blog_link: Blog
+top: Voltar ao início
+
+#---------
+# Articles
+#
+#   Cada artigo tem estatísticas na parte inferior como esta:
+#   934.8 mil visualizações (↑300.5 mil em relação a ontem)
+#------------------------------------------------------------------------------
+
+views: visualizações
+from_yesterday: desde ontem
+streak: sequência de dias
+
+#------------
+# About section
+#------------------------------------------------------------------------------
+
+about_header: Sobre
+about_section: >-
+     <p>Leia o <a href="http://top.hatnote.com/en/wikipedia">arquivo</a>.</p>
+
+     <p>Uma lista diária dos artigos mais visitados da Wikipédia, gerada
+     por <a href="https://twitter.com/sklaporte">Stephen LaPorte</a>
+     and <a href="https://twitter.com/mhashemi">Mahmoud
+     Hashemi</a>. Também está disponível em <a
+     href="http://top.hatnote.com/about.html">outros idiomas</a>. Se você achou esta lista interessante,
+     vai gostar do
+     <a href="http://weekly.hatnote.com">The Weeklypedia</a>, um resumo semanal dos artigos mais editados da Wikiédia. Saiba mais no <a
+     href="#">Hatnote Blog</a>.</p>
+
+     <p>Esta página foi possível graças à <a
+     href="https://wikimedia.org/api/rest_v1/?doc#!/Pageviews_data/get_metrics_\
+     pageviews_top_project_access_year_month_day">Nova API de visualizações de páginas da Wikipedia</a>,  com algumas imagens e resumos fornecidos pela <a
+     href="https://www.mediawiki.org/wiki/API:Query">API da MediaWiki</a>.
+     Você pode encontrar mais código e documentação
+     <a href="https://github.com/hatnote/top">disponíveis no github</a>.</p>
+
+     <p>Quer ver uma lista dos artigos mais visitados em outro idioma da Wikipédia? Nós também! Por favor, <a
+     href="https://github.com/hatnote/top/issues">envie uma solicitação</a>
+     e nos avise se puder ajudar a traduzir os templates.</p>
+
+     <p><a href="https://twitter.com/hatnotable">@hatnotable</a></p>
+     <p>Leia o <a href="http://top.hatnote.com/en/wikipedia">arquivo</a>.</p>
```

Comment:
This line is at the top and bottom. Probably just need the one at the top.

Reply:
@Vazafirst Not a big deal, but this line appears once in en_strings's About section, whereas it's repeated twice in the new pt_strings About section (top and bottom).